### ISSUE TITLE
Getting already connected devices

### ISSUE THREAD

**Issue by gnichs** [Arti3DPlayer/USBDeviceSwift#1]
Hi Arti,

Do you plan to add support for getting already connected USB/HID devices?

Thanks,
Graham

**Comment by Arti3DPlayer** [comment]
@gnichs With global notifications 
```
USBDeviceConnected
USBDeviceDisconnected
```
you can easy control your list of devices. When program starts it received this events from already connected devices, so you can handle them

**Comment by gnichs** [comment]
Thanks! You're totally right. Looks like I was just using it incorrectly. 

By the way, I noticed a couple small typos in your Getting Started documentation regarding getting a disconnected device's id. The method should be called `usbDisconnected` rather than `usbConnected` and the second guard statements should read 

```
guard let id:UInt64 = nobj["id"] as? UInt64.
```
instead of 

```
guard let deviceInfo:USBDevice = nobj["id"] as? UInt64 else
```

I noticed that I had to correct them in order to get my app to compile. I figured pointing it out could help other get started a bit easier.

Thanks again for your help!
Graham
